azurerm/internal/services/iotcentral/resource_arm_iotcentral_aplication.go

```diff
@@ -0,0 +1,257 @@
+package iotcentral
```

Comment:
file name should be resource_arm_iotcentral_application, right?

Reply:
@xuzhang3   mispell application name